### PR TITLE
Improve config and reduce fulltest

### PR DESCRIPTION
**Pull request by LouisLeNezet**
The truth variants set of NA12878 is now the one computed by [Genome in a Bottle](https://ftp-trace.ncbi.nlm.nih.gov/ReferenceSamples/giab/release/).

## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/phaseimpute/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/phaseimpute _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core pipelines lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Check for unexpected warnings in debug mode (`nextflow run . -profile debug,test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: conf/test_full.config [hunk 1]
```diff
@@ -40,5 +41,5 @@ params {
     phase          = false
 
     // Impute tools
-    tools = "glimpse1"
+    tools = "glimpse2"
```
Comment (atrigila) [inline conf/test_full.config:44]: Why this change? I am confused as the reason behind this.

Comment (atrigila) [reply conf/test_full.config:44]: I mean, we are going to launch a completely different AWS mega test from the one before.

Comment (LouisLeNezet) [reply conf/test_full.config:44]: The aim is to reduce the computational cost of the megatest as much as possible.
The `glimpse2` software as less steps (no variant calling) and should also be more computationally efficient.
I'll add it to the changelog

**Review by atrigila — APPROVED** [review]
While I may have some reservations about these changes, I also don’t want to block progress toward the release. I trust your verification that the full `glimpse2` megatest works, that it is less computationally expensive, and aligns with the new genome in a bottle VCF :)

**Comment by LouisLeNezet** [comment]
I've checked on my cluster on a subset and everything worked perfectly ;)
